azure-nextgen-py-static-website/__main__.py

```diff
@@ -0,0 +1,111 @@
+import pulumi
+import pulumi_azure_nextgen as azure_nextgen
+import pulumi_random as random
+
+# TODO: Remove after autonaming support is added.
+random_suffix = random.RandomString("randomSuffix",
+    length=10,
+    special=False,
+    upper=False)
+
+config = pulumi.Config()
+location = config.get("location")
```

Comment:
0.6.1 has auto-location population. So, we don't need this config: instead, we should suggest using `pulumi config set azure-nextgen:location westus`.